plugins/base/src/main/resources/dokka/styles/prism.css

```diff
@@ -64,25 +62,28 @@ https://prismjs.com/download.html#themes=prism&languages=clike+java+javadoclike+
 .style .token.string {
 	color: #9a6e3a;
 	/* This background color was intended by the author of this theme. */
-	/*background: hsla(0, 0%, 100%, .5);*/
+	background: hsla(0, 0%, 100%, 0.5);
 }
 
 .token.atrule,
 .token.attr-value,
 .token.keyword {
-	color: var(--keyword-color);
```

Comment:
Why do you not use CSS variables? It is more convenient when there are several themes.

Reply:
I didn't write it out by hand, it's a direct copy-paste from webhelp's PR, so you best address that question to them :)

I could go out of my way and substitute colors with variables, but I'd prefer to keep changes to a minimum (and document them as I did in the header) so that it's easier to update it in the future. 

That was actually the whole reason why I needed to copy all of the styles instead of making selective changes.

Reply:
It is the easiest way, but I am not sure WH styles will be consistent with Dokka further.

Reply:
Yeah, good point :+1:  Although if we do change any of the copy-pasted code from webhelp, during updates it'll be very difficult to tell which styles were copy-pasted and should be replaced, and which were custom.

So there's a need for Dokka-specific style changes, I think it should be done as overrides at the bottom of the file. This way it can be both easily kept in sync with upstream and allow for some custom Dokka colors